.github/workflows/python-ci.yml

```diff
@@ -0,0 +1,44 @@
+name: Python CI
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    name: Lint with PyLint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+
+      - name: Install dependencies
+        run: |
+          pip install --upgrade pip
+          pip install -r requirements.txt
+          pip install pylint
+
+      - name: Run PyLint
+        run: pylint . --ignore=venv --disable=C0103,W0221,W0621
```

Comment:
@filipp05  These exceptions must be excluded as well. 
If needed it must be placed right in the place of exception.